CHANGELOG.md

```diff
@@ -14,4 +14,8 @@ Initial release.
 ### Added
 - OrchardBundle enum
 - first_issuance flag
-- action-group-based serialization
\ No newline at end of file
+- action-group-based serialization
+
+## [Unreleased] - 2025-05-05
+### Added
+- Support for scenarios with arbitrary number of accounts
```

Comment:
Update with more details and move all `[Unreleased]' items to `[0.3.0]`.
We are going to release after this is merged.

Reply:
Added further details